Hippo.Web/ClientApp/src/components/Admin/AdminUsers.tsx

```diff
@@ -117,17 +119,17 @@ export const AdminUsers = () => {
               </tr>
             </thead>
             <tbody>
-              {users.map((user) => (
-                <tr key={user.id}>
-                  <td>{user.name}</td>
-                  <td>{user.email}</td>
+              {accounts.map((account) => (
+                <tr key={account.id}>
+                  <td>{account.owner?.name}</td>
+                  <td>{account.owner?.email}</td>
                   <td>
                     <button
                       disabled={notification.pending}
-                      onClick={() => handleRemove(user)}
+                      onClick={() => handleRemove(account)}
                       className="btn btn-primary"
                     >
-                      {adminRemoving === user.id ? "Removing..." : "Remove"}
+                      {adminRemoving === account.id ? "Removing..." : "Remove"}
```

Comment:
I think this still needs to be the user id? account.owner.id

Reply:
not sure but we probably want to move to account ids and away from owner useage

Reply:
@srkirkland This was my mistake. This needs to be account.id. But I had to change the post to use user.id.
Fixed in my pr